packages/client/tests/e2e/ts-version/5.1/package.json

```diff
@@ -10,6 +10,6 @@
     "@types/jest": "29.5.1",
     "@types/node": "16.18.11",
     "prisma": "/tmp/prisma-0.0.0.tgz",
-    "typescript": "~5.1.6"
+    "typescript": "~5.2.2"
```

Comment:
`5.1` and `5.2` are mixed up, `5.1` has `"typescript": "~5.2.2"` and `5.2` has `"typescript": "~5.1.6"`

Reply:
Nice 🙈 
Thanks!